gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Fri Nov 25 11:23:49 CET 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.2-bin.zip
```

Comment:
Did you use the assistant for the migration? i'm surprised to see only the properties file changed

Reply:
Yes I did, even took a screenshot before doing so:

<img width="990" alt="Capture d’écran 2023-12-19 à 13 54 55" src="https://github.com/didomi/sample-app-android-java/assets/102527376/e655ae37-b6bf-4283-b778-591dd0be9b3d">